_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+newcomers as well as developers to know more about this community and about
+what they have contributed to it.
+
+### Goals
+
+* Add calendar app to show timeline for various events
+* Add a one-stop tab for all google forms
+* Add contributors geolocation
+* Add *Login/SignUp* using GitHub
+* Add organization teams
+* Advanced filters
+* Automate welcoming process for newcomer
+* Automate newcomer promotion
+* Create new database models to manage data
+* Ease navigatibility in websites
+* Improve the **UI/UX of websites** to make more interactive
+* Motivating newcomers & developers with -
+  * modifications in Gamification Leaderboard
+  * adding a tab of **GSoC** in projects website
+* New coala-community logo
+* Contributors profile page
+* Admin/Maintainers dashboard
+
+### Milestones
+
+#### PREPARATION/BONDING
+
+* Detailed research on what **new cron jobs** or **API endpoints** need to be
+  added in [coala webservices](https://gitlab.com/coala/landing/).
+* Discussion on improvements to be made in coala [community website](http://community.coala.io/) and
+  [projects website](http://projects.coala.io/)
+* Research on new models to be created (if any) and adding new fields to
+  existing db models in coala community website **db models**.
+* *Created* and *Merged* a cEP of how the new enhancements will be implemented
+  and how will it be beneficial to [coala community](https://github.com/orgs/coala/teams/coala-newcomers).
+
+#### CODING PHASE 1
+
+1. Implementation of methods in **coala [webservices](https://gitlab.com/coala/landing)**
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create new **[Community Logo](https://github.com/coala/community/issues/55) + Header Image of README.md**
+
+    b. Add information about new endpoints at coala-webserivces homepage.
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
+
+    d. Add [Organisation Teams](https://github.com/coala/community/issues/12)
+
+    e. Add [Mentors](https://github.com/coala/community/issues/11)
+
+    f. Add **Newcomer Invitation** [form](https://github.com/coala/community/issues/89#issuecomment-468955279)
+
+    g. Add **Promotion to Developer** [form](https://github.com/coala/community/issues/89)
+
+    h. Manage live data of community members
+
+    i. Addition of more data fields in **Users** db model
+
+    j. Add **Login and Sign-up** using GitHub
+3. Make Improvements (or fix bugs) in coala [community website](https://community.coala.io/)
+
+    a. Build failure for more than one remote [#235](https://github.com/coala/community/issues/235)
+
+    b. Netlify build failure for fork [#232](https://github.com/coala/community/issues/232)
+
+#### CODING PHASE 2
+
+1. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create **Profile Page**
+
+    b. Leaderboard on **[Homepage](https://coala-community.netlify.com/)**
```

Comment:
Why do we need to have the leaderboard on the homepage?

Reply:
Not complete leaderboard, just showing only top 10 (or any other number) with only usernames. And, that too can be filtered by week, month and year.
Adding on the homepage will make the website more interactive and creative. Also, it will give users an idea that we have a `gamification system` implemented and they can see full `gamification leaderboard` either by `navigating through the menu` or by clicking on `see more results` under the leaderboard shown on the homepage.

Reply:
Sounds cool. But adding only `gamification leaderboard` on the home page makes it look like that is only the important thing in this website. There are other links also on the home page right now, E.g. Meta-review data, contributors-data, google-code-in etc.. imo have a clickable card for each link on the home page..and then by clicking on the leader-board user will be redirected to the leaderboard dashboard which is designed in a cool way and have all the functionalities that you're saying..similar cases for other cards...what do you think?

Reply:
Yep, I like your idea. We can have clickable cards for each link on the homepage which will display some important information and then upon clicking it the user will be redirected to the original web page. I will think upon it, on how can we make homepage better with this idea i.e. what all can be displayed for other links in that card and will it need some additions or not!

Reply:
Should I add the website templates or designs as well as colour palette about the new look of websites? Because it will take time to create designs using photoshop or searching for best designs on internet.